application-templates/starter/package.json

```
@@ -53,7 +53,7 @@
     "eslint": "8.7.0",
     "eslint-formatter-pretty": "4.1.0",
     "jest": "27.4.7",
-    "jest-runner-eslint": "0.11.1",
+    "jest-runner-eslint": "^1.0.0",
```

Let's pin it. We have bots running so no need to have it flexible.

```suggestion
    "jest-runner-eslint": "1.0.0",
```

---

Strange I am not getting the error locally. You're sure all versions of the dependency didn't have the caret?

---

Yes, you're right. I reviewed my first commit and I messed with the caret in one of the package.json files.

I've just update both of them to use a fixed version.

---

Lovely, thanks a lot!